tokens/src/core/global/typography.json

```diff
@@ -5,72 +5,77 @@
         "base": {
           "value": "{typography.font.family.sans.serif}",
           "type": "fontFamily",
-          "source": "$font-family-base"
+          "source": "$font-family-base",
+          "description": "Basic font family."
         },
         "sans": {
           "serif": {
             "value": "-apple-system, BlinkMacSystemFont, \"Segoe UI\",  Roboto, \"Helvetica Neue\", Arial, \"Noto Sans\", sans-serif, \"Apple Color Emoji\",  \"Segoe UI Emoji\", \"Segoe UI Symbol\", \"Noto Color Emoji\"",
             "type": "fontFamily",
-            "source": "$font-family-sans-serif"
+            "source": "$font-family-sans-serif",
+            "description": "Sans-serif font family."
           }
         },
-        "serif": { "value": "serif", "source": "$font-family-serif" },
+        "serif": { "value": "serif", "source": "$font-family-serif", "description": "Serif font style" },
```

Comment:
the other ones say "font family" but this one stays "font style"

Reply:
Changed to `Serif font family.`